src/components/BaseCanvas/BaseCanvas.component.tsx

```diff
@@ -27,7 +29,13 @@ export const BaseCanvas: FC<BaseCanvasProps> = ({
     key={fov}
     className={`${styles['base-canvas']} ${className ?? ''}`}
     shadows="soft"
-    gl={{ preserveDrawingBuffer: true, alpha: true, toneMappingExposure: 1.6, toneMapping: ACESFilmicToneMapping }}
+    gl={{
+      preserveDrawingBuffer: true,
+      alpha: true,
+      toneMapping: ACESFilmicToneMapping,
+      toneMappingExposure: 1.8
+    }}
+    flat={enablePostProcessing}
```

Comment:
Since the color handling was revamped with the new three version we toggle flat color handling on manually if post processing is enabled. Another adjustment is the slight tone mapping exposure bump from 1.6 to 1.8 in order to align things.